server/middleware/errorsMiddleware.js

```diff
@@ -0,0 +1,12 @@
+const logger = require('../../log.js')
+const { GENERAL_ERROR } = require('../helpers/errorConstants')
+
+// eslint-disable-next-line no-unused-vars
+const errorsMiddleware = (error, req, res, next) => {
+  logger.error(error)
+  const { stack, status = 500, userMessage: message = GENERAL_ERROR } = error
+  res.status(status)
+  res.render('pages/error', { stack: process.env.NODE_ENV === 'production' ? null : stack, message, status })
```

Comment:
Are we only removing the error messages in production?

Reply:
'production' is dev,preprod,prod and not local. not arguing for one or the other, just sayin'

Reply:
Thats mostly an existing bit of code. The stack is not shown on any env where we set the NODE_ENV to 'production'. If we want to readdress our approach to this I'd suggest we look at it separately? Theres already too many moving parts in this PR.

Reply:
I am personally for us having the same view in all environments. I frequently run branches locally to test my changes/code review and the difference in running locally has thrown up numerous issues for me on this project